Maths/FriendlyNumbers.js

```diff
@@ -0,0 +1,41 @@
+/*
+  'In number theory, friendly numbers are two or more natural numbers with a common abundancy index, the
+  ratio between the sum of divisors of a number and the number itself.'
+  Source: https://en.wikipedia.org/wiki/Friendly_number
+  See also: https://mathworld.wolfram.com/FriendlyNumber.html#:~:text=The%20numbers%20known%20to%20be,numbers%20have%20a%20positive%20density.
+*/
+
+export const FriendlyNumbers = (firstNumber, secondNumber) => {
+  // input: two integers
+  // output: true if the two integers are friendly numbers, false if they are not friendly numbers
+
+  // First, check that the parameters are valid
+  if (!Number.isInteger(firstNumber) || !Number.isInteger(secondNumber) || firstNumber === 0 || secondNumber === 0 || firstNumber === secondNumber) {
+    throw new Error('The two parameters must be distinct, non-null integers')
+  }
+
+  // Calculate the abundancy index of the two number.
```

Comment:
could write a function to do that

Reply:
Absolutely! Thanks for the advice. I commited the updated code. 
Please let me know if something is amiss (I'm completely new to gitHub and git....)